enrich-stream/enrich-stream.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2019 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+import json
+import base64
+import os
+
+
+def init_context(context):
+    # env -> config
+    setattr(context.user_data, 'config', {
+        'v3io_api': os.environ['V3IO_API'],
+        'v3io_username': os.environ['V3IO_USERNAME'],
+        'container_name': os.environ['CONTAINER_NAME'],
+        'table_name': os.environ['TABLE_NAME'],
+        'input_stream_search_key': os.environ['INPUT_STREAM_SEARCH_KEY'],
+        'output_stream_name': os.environ['OUTPUT_STREAM_NAME'],
+        'v3io_access_key': os.environ['V3IO_ACCESS_KEY'],
+    })
+
+
+def handler(context, event):
+    config = context.user_data.config
+    msg = json.loads(event.body)
+    context.logger.info(f'Incoming message: {msg}')
```

Comment:
use logging structure, as follows: (separate variables from static log message)
```
context.logger.info('Incoming message', msg=msg)
```
(relevant for the other logs too)